verification/curator-service/api/src/controllers/auth.ts

```diff
@@ -306,25 +306,25 @@ export class AuthController {
                     }
                     // prefix the API key with the user ID to make it easier to find users by API key in auth
                     const randomPart = await getRandomString(32);
-                    currentUser.apiKey = `${theUser.id}${randomPart}`;
+                    currentUser.apiKey = `${theUser.id.toString()}${randomPart}`;
```

Comment:
This is the one meaningful change in this commit. The rest was done by the husky pre-commit hook :-/